packages/extension/src/view/devtools/stateProviders/filterManagementStore/utils/matchTerm.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies.
+ */
+import type { CookieTableData } from '../../../cookies.types';
+
+/**
+ * Matches the search term with the cookie name and domain and adds it to the filtered cookies object.
+ * @param searchTerm The search term.
+ * @param cookieData The cookie data.
+ */
+export default function matchTerm(
+  searchTerm: string,
+  cookieData: CookieTableData,
+  filteredCookies: { [key: string]: CookieTableData }
+) {
+  const cookieName = cookieData.parsedCookie.name;
+
+  if (!searchTerm) {
+    filteredCookies[cookieName] = cookieData;
+  }
+
+  const lowerCaseTerm = searchTerm.toLowerCase();
+  const didSearchTermMatch =
```

Comment:
@sayedtaqui Did we plan to only search through name and domain?

Reply:
As of now yes, any suggestions?